public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -43,7 +43,13 @@ export function setupKeyboardShortcuts(scene: DashboardScene) {
   keybindings.addBinding({
     key: 'v',
     onTrigger: withFocusedPanel(scene, (vizPanel: VizPanel) => {
-      if (!scene.state.viewPanelScene) {
+      if (scene.state.viewPanelScene) {
+        locationService.push(
+          locationUtil.getUrlForPartial(locationService.getLocation(), {
+            viewPanel: undefined,
+          })
+        );
```

Comment:
```ts
        locationService.push(
          locationUtil.getUrlForPartial(locationService.getLocation(), {
            viewPanel: undefined,
          })
        );
```
I wonder if we could factor out the snippet above into a function? So something like
```js
function updateQueryParams(params) {
    locationService.push(locationUtil.getUrlForPartial(locationService.getLocation(), params));
}
```

and then we could use that for `{editPanel: undefined}` as well.